examples/monitoring/openai_ux_demo.ipynb

```diff
@@ -8,11 +8,12 @@
     "# OpenAI LLM Monitoring with W&B Weave\n",
     "\n",
     "Use the W&B OpenAI integration to monitor OpenAI API calls and understand how your projects and teams are leveraging LLMs.\n",
-    "This example notebook will generate links to Weave Boards: LLM usage monitoring dashboards which you can explore and customize from the UI.\n",
+    "In this example, we'll setup a LLM usage monitoring dashboards which you can explore and customize from the UI.\n",
     " \n",
+    "* track cost, latency and token counts to see how your organization are using LLMs\n",
+    "* manage permissions and hosting to ensure data privacy\n",
```

Comment:
took a pass at a compromise, I really don't think this "manage permissions and hosting to ensure data privacy" is true/possible right now.